Core/Utilities/TimeLeft/TimeLeft.py

```diff
@@ -215,4 +202,31 @@ def runCommand( cmd, timeout = 120 ):
     return S_OK( str( stdout ) )
 
 
+def enoughTimeLeft(cpu, cpuLimit, wallClock, wallClockLimit, cpuMargin, wallClockMargin):
+  """ Is there enough time?
+
+      :returns: True/False
+  """
+
+  cpuRemainingFraction = 1. - cpu / cpuLimit
+  wallClockUsedFraction = wallClock / wallClockLimit
```

Comment:
You can simplify as above as wallClockUsedFraction is not used
I would also multiply these 2 by 100 and removed 214,215 and the 100. * in 216
This is cosmetics but while there ;-)